js/pro/binance.js

```diff
@@ -71,7 +71,15 @@ module.exports = class binance extends binanceRest {
                 'wallet': 'wb', // wb = wallet balance, cw = cross balance
                 'listenKeyRefreshRate': 1200000, // 20 mins
                 'ws': {
-                    'cost': 5,
+                    'rateLimits': {
```

Comment:
@pcriadoperez two things,

As discussed, should we require the `rateLimit` only and infer the other values?

Second, since we end up having a RL for the connections, should we allow a specific config for it? and if not found use the default?

Reply:
@carlosmiei I added the 'newConnections' config. You think it's ok if we merge this for now and then in the future we can come up with a good formula for `rateLimit`?

Reply:
@pcriadoperez 

> What do you think is a good formula to infer the values from the rateLimit?

you can check how the rest Exchange does it

> Yes, that makes sense, whats a good name `['ws']['rateLimits']['newConnections'] ?

New connections is fine I think, but if they're exactly the same we don't need to add it right?